src/IECore/FileSequenceFunctions.cpp

```diff
@@ -42,9 +42,9 @@
 #include "IECore/ReversedFrameList.h"
 
 #include "boost/algorithm/string.hpp"
-#include "boost/filesystem/convenience.hpp"
-#include "boost/filesystem/operations.hpp"
 #include "boost/filesystem/path.hpp"
+#include "boost/filesystem/operations.hpp"
+#include "boost/filesystem/directory.hpp"
```

Comment:
Minor nitpick : we order includes alphabetically, so `directory.hpp` would come before `operations.hpp`, which would come before `path.hpp`.

Reply:
yes. it is fixed in the next commit.